websites/S/Sushi Scan/metadata.json

```
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.12",
+	"apiVersion": 1,
+	"author": {
+		"id": "723471302123323434",
+		"name": "redstom"
+	},
+	"service": "Sushi Scan",
+	"description": {
+		"en": "Read comics on Sushi Scan.",
+		"fr": "Lisez des mangas et bandes dessinées sur Sushi Scan."
+	},
+	"url": [
+		"sushiscan.net",
+		"www.sushiscan.net"
+	],
+	"version": "1.0.0",
+	"logo": "https://sushiscan.net/wp-content/uploads/IconSushiScan.png",
```

This URL might not work, since the site returns a captcha before directing to the image
```suggestion
	"logo": "https://cdn.discordapp.com/app-assets/1324071536017149973/1324137722079875112.png?size=512",
```